lib/whatsapp_sdk/api/messages.rb

```diff
@@ -363,9 +363,52 @@ def send_contacts(
       #   # TODO: https://developers.facebook.com/docs/whatsapp_sdk/cloud-api/reference/messages#contacts-object
       # end
 
-      # def send_interactive_reply_buttons
-      #   # TODO: https://developers.facebook.com/docs/whatsapp_sdk/cloud-api/reference/messages#contacts-object
-      # end
+      # Send interactive reply buttons.
+      # https://developers.facebook.com/docs/whatsapp/guides/interactive-messages#reply-buttons
+      # You can either send interactive object or as JSON.
+      #
+      # @param sender_id [Integer] Sender' phone number.
+      # @param recipient_number [Integer] Recipient' Phone number.
+      # @param interactive [Interactive] Interactive.
+      # @param interactive_json [Json] The interactive as a Json. If you pass interactive_json, you can't pass interactive.
+      # @param message_id [String] The id of the message to reply to.
+      # @return [WhatsappSdk::Api::Response] Response object.
+      sig do
+        params(
+          sender_id: Integer, recipient_number: Integer, interactive: T.nilable(T.untyped),
+          interactive_json: T.nilable(String), message_id: T.nilable(String),
```

Comment:
Could `interactive_json` be a JSON type instead? 

`T::Hash[T.untyped, T.untyped]` ?